components/ILIAS/Mail/classes/class.ilMimeMail.php

```diff
@@ -252,10 +267,15 @@ private function removeHTMLTags(string $maybeHTML): string
         return strip_tags($maybeHTML);
     }
 
-    protected function buildBodyMultiParts(string $skin, string $style): void
+    private function getPathToRootDirectory(): string
+    {
+        return realpath(dirname(__DIR__, 4) . '/');
```

Comment:
I suggest making this more dynamic. We could traverse up the directory tree until we find the `ilias_version.php`. I agree, even this is a bit fragile, but as long as the ILIAS core does not provide a central helper function for this, this is IMO the best approach.